boards/arm/reel_board/Kconfig.defconfig

```diff
@@ -41,20 +41,20 @@ endif # I2C_1
 
 endif # I2C
 
-if SPI
+if PWM
 
-config SPI_3
+config PWM_0
 	default y
 
-config SPI_NRFX_RAM_BUFFER_SIZE
-	default 8
```

Comment:
Was this intentional, i.e. transfers from flash are not supposed to occur for `spi_3` (`arduino_spi`)?
Maybe it would be safer to keep this feature enabled by default? And if it is not needed for a given application, it can be disabled there, to save a bit of RAM and flash, and some execution cycles.

Reply:
@anangl You are probably right, I will check

Reply:
Followed your suggestion, thanks